euth/organisations/models.py

```diff
@@ -1,7 +1,9 @@
 from django.conf import settings
 from django.db import models
+
 from model_utils import models as model_utils
 
+from ..contrib import validators
```

Comment:
I would advise to use relative imports only inside an app. Otherwise absolute imports are better; in this case `from euth.contrib import validators`. It makes the apps independent from the overall project structure which simplifies a future extraction of the app.